fiftyone/core/annotation/generate_label_schemas.py

```
@@ -363,8 +367,7 @@ def _generate_field_label_schema(collection, field_name, scan_samples):
                 collection, f"{field_name}.{f.name}", scan_samples
             )
         except ValueError:
-            # Field type not supported for schema generation
-            pass
+            logger.debug(f"Field '{f.name}' is not supported")
```

Ruff (0.14.14)

[warning] Logging statement uses f-string

(G004)